run_tests

```diff
@@ -0,0 +1,137 @@
+# Note that this script is taken from raven "raven/run_tests". Some changes were made to make
```

Comment:
this note should probably be finished or removed

Reply:
Fixed it.